lib/flutter_redux.dart

```diff
@@ -442,76 +453,75 @@ class _StoreStreamListenerState<S, ViewModel>
 
   @override
   void didUpdateWidget(_StoreStreamListener<S, ViewModel> oldWidget) {
+    latestValue = widget.converter(widget.store);
+
     if (widget.store != oldWidget.store) {
-      _init();
+      _createStream();
```

Comment:
I see that now we don't call the onInit/onInitialBuild anymore again on `didUpdateWidget`, this could be a breaking change for someone (although maybe unlikely?)

Also, after commenting out this statement I didn't see any test failing. I think there's no test covering what happens if a user changes a store. Also unlikely as I think having one single Store is the common approach. But to keep in mind.

Reply:
Thanks! That's actually a fair point and a hole in the test coverage. 

I think the change makes sense overall since it really is only meant to be called when the Widget is first initialized, but I'll add tests for this case.